scripts/docker/hub/Dockerfile

```diff
@@ -31,6 +31,8 @@ COPY artifacts/x86_64-unknown-linux-gnu/$TARGET ./bin/$TARGET
 RUN echo "#!/bin/bash \n ${TARGET} \$@" > ./entrypoint.sh
 RUN chmod +x ./entrypoint.sh
 
+ADD check_sync.sh /check_sync.sh
```

Comment:
missing a leading dot here ./check_sync.sh

Reply:
The script should still be executable and I would prefer to invoke it from `/check_sync.sh` rather than `/home/parity/check_sync.sh`

Either works though, the latter is just more cumbersome.